src/commands/branches.ts

```diff
@@ -23,15 +23,15 @@ import { parseSchemaDiffParams, schemaDiff } from './schema_diff.js';
 const BRANCH_FIELDS = [
   'id',
   'name',
-  'primary',
+  'default',
   'created_at',
   'updated_at',
 ] as const;
 
 const BRANCH_FIELDS_RESET = [
   'id',
   'name',
-  'primary',
+  'default',
```

Comment:
Given we support outputting in "json" and "yaml" format, not sure if any user further uses this output/field. Maybe momentarily, we should keep both the "default" and "primary" fields in the output fields?

Reply:
Kept `primary` in both `BRANCH_FIELDS` and `BRANCH_FIELDS_RESET`.